plugins/BEdita/Core/src/Model/Entity/ObjectType.php

```diff
@@ -377,6 +379,39 @@ protected function _getSchema()
         return $event->getResult() ?: $schema;
     }
 
+    /**
+     * Virtual property to get info about permissions availability.
+     *
+     * @return bool
+     */
+    protected function _getPermissionsEnabled(): bool
```

Comment:
Instead of a virtual prop can we use methods like 
`hasAssoc(string $name)` and `addAssoc(string $name)` `removeAssoc(string $name)` ? 

This way we don't expose a new property in API calls and we don't need to hide it if we don't want it

The permissions enabled/disabled information is already in the `associations` property

Reply:
:thinking: I like to have in `meta` an explicit information about permission enabled/disabled rather than having to figure it out from associations.

Reply:
After some thinking I'm going to remove `permissions_enabled` virtual property as suggested.